libecho/include/EchoCalibration.h

```diff
@@ -6,10 +6,27 @@
  */
 class EchoCalibration {
 public:
+    /**
+     * @brief Returns an instance
+     * @param win_size Number of samples per bit
+     * @param freq     The frequency
+     */
```

Comment:
kropka
Ja bym też dodała że to frequency i win_size, dla których chcemy znaleźć lim.